src/inputs/getPullRequestNumber.ts

```diff
@@ -58,7 +58,7 @@ async function getPullRequestNumber(
 		const sha = github.context.payload.workflow_run.head_sha;
 		// ... in all other cases, we have to call the API to get a matching PR number
 		core.info(
-			`Trying to find a pull-request with a head commit matchin the SHA found in the action's "payload.workflow_run.head_sha" context (${sha}) from the GitHub API.`,
+			`Trying to find a pull-request with a head commit matching the SHA found in the action's "payload.workflow_run.head_sha" context (${sha}) from the GitHub API.`,
```

Comment:
This is the only change that impacts the generated `dist/` directory.

Until https://github.com/davelosert/vitest-coverage-report-action/pull/544/files#r2484915536 there isn't a way to generate the `dist/` contents, but realistically, it makes no sense for anyone to actually include those changes in their PRs as they should be regenerated by a maintainer.

I have a version of the changes for this file, but it would conflict with the other PR, so I haven't included it.